yt/units/yt_array.py

```diff
@@ -22,14 +22,20 @@
     add, subtract, multiply, divide, logaddexp, logaddexp2, true_divide, \
     floor_divide, negative, power, remainder, mod, absolute, rint, \
     sign, conj, exp, exp2, log, log2, log10, expm1, log1p, sqrt, square, \
-    reciprocal, ones_like, sin, cos, tan, arcsin, arccos, arctan, arctan2, \
+    reciprocal, sin, cos, tan, arcsin, arccos, arctan, arctan2, \
     hypot, sinh, cosh, tanh, arcsinh, arccosh, arctanh, deg2rad, rad2deg, \
     bitwise_and, bitwise_or, bitwise_xor, invert, left_shift, right_shift, \
     greater, greater_equal, less, less_equal, not_equal, equal, logical_and, \
     logical_or, logical_xor, logical_not, maximum, minimum, fmax, fmin, \
     isreal, iscomplex, isfinite, isinf, isnan, signbit, copysign, nextafter, \
     modf, ldexp, frexp, fmod, floor, ceil, trunc, fabs, spacing
 
+try:
+    # numpy 1.13 or newer
+    from numpy import positive, divmod as divmod_, isnat, heaviside
+except ImportError:
+    positive, divmod_, isnat, heaviside = (None,)*4
```

Comment:
No, I'm setting these symbols to `None` since they're not available in older versions of Numpy